### PR TITLE
ok this really has very little to do with Entity AI stuff at this point

### PR DESCRIPTION
**Pull request by Prospector**

### CODE REVIEW

File: mappings/net/minecraft/entity/ai/goal/Goal.mapping [hunk 1]
```diff
@@ -2,6 +2,7 @@ CLASS aio net/minecraft/entity/ai/goal/Goal
 	FIELD a categoryBits I
 	METHOD a canStart ()Z
 	METHOD a setCategoryBits (I)V
+		ARG 1 categoryBits
```
Comment (liach) [inline mappings/net/minecraft/entity/ai/goal/Goal.mapping]: Should be control bits instead

Comment (Prospector) [reply mappings/net/minecraft/entity/ai/goal/Goal.mapping]: Oh ok

File: mappings/net/minecraft/entity/ai/goal/Goal.mapping [hunk 2]
```diff
@@ -1,11 +1,11 @@
 CLASS aio net/minecraft/entity/ai/goal/Goal
-	FIELD a categoryBits I
+	FIELD a controlBits I
 	METHOD a canStart ()Z
 	METHOD a setCategoryBits (I)V
```
Comment (liach) [inline mappings/net/minecraft/entity/ai/goal/Goal.mapping]: you forgot to rename this method

Comment (Prospector) [reply mappings/net/minecraft/entity/ai/goal/Goal.mapping]: interesting...I swear I did :P

**Comment by liach** [comment]
lgtm

**Comment by asiekierka** [comment]
You missed BlockEntity.getFactory()

**Comment by asiekierka** [comment]
LGTM

File: mappings/net/minecraft/item/AxeItem.mapping [hunk 3]
```diff
@@ -1,4 +1,5 @@
 CLASS atr net/minecraft/item/AxeItem
+	FIELD a STRIPPED_LOGS_MAP Ljava/util/Map;
```
Comment (asiekierka) [inline mappings/net/minecraft/item/AxeItem.mapping]: ShovelItem has a very similar map. I would call this something more generic as such, such as "BLOCK_TRANSFORMATIONS_MAP"

Comment (Prospector) [reply mappings/net/minecraft/item/AxeItem.mapping]: Changed

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**